modules/geth/charts.go

```diff
@@ -0,0 +1,143 @@
+package geth
+
+import "github.com/netdata/go.d.plugin/agent/module"
+
+type (
+	Charts = module.Charts
+	Chart  = module.Chart
+	Dims   = module.Dims
+	Dim    = module.Dim
+)
+
+var charts = Charts{
+	chartAncientChainData.Copy(),
+	chartChaidataDisk.Copy(),
+	chartTxPoolPending.Copy(),
+	chartBlockProcessingTime.Copy(),
+	chartTxPoolQueued.Copy(),
+	chartP2PNetwork.Copy(),
+	chartP2PNetworkDetails.Copy(),
+	chartNumberOfPeers.Copy(),
+	chartRpcInformation.Copy(),
+}
+
+var (
+
+	chartAncientChainData = Chart{
+		ID:    "chaindata_ancient",
+		Title: "Ancient Chaindata",
+		Units: "bytes",
+		Fam:   "chaindata",
+		Ctx:   "geth.chaindata",
+		Dims: Dims{
+			{ID:ethDbChainDataAncientRread , Name: "Ancient chaindata reads"},
+			{ID: ethDbChainDataAncientWrite, Name: "Ancient chaindata writes", Mul: -1},
+		},
+		
+	}
+	chartChaidataDisk = Chart{
+		ID:    "chaindata_disk",
+		Title: "Chaindata on disk",
+		Units: "bytes",
+		Fam:   "chaindata",
+		Ctx:   "geth.chaindata_disk",
+		Dims: Dims{
+			{ID:ethDbChaindataDiskRead , Name: "Disk Chaindata reads"},
+			{ID: ethDbChainDataDiskWrite, Name: "Disk Chaindata writes", Mul: -1},
+		},
+	}
+	chartBlockProcessingTime = Chart{
+		ID: "blockProcessing_time", 
+		Title: "Block processing time",
+		Units: "seconds",
+		Fam: "block_processing", 
+		Ctx: "geth.block_processing",
+		Dims: Dims{
+			{ID: blockProcessing, Name: "Block processing Time"},
+		},
+	}
+	chartTxPoolPending = Chart{
+		ID: "txpoolpending",
+		Title: "Pending Transaction Pool",
+		Units: "transactions", 
+		Fam: "tx_pool",
+		Ctx: "geth.tx_pool_pending", 
+		Dims: Dims{
+			{ID: txPoolInvalid, Name: "Invalid transaction pool"},
+			{ID: txPoolPending, Name: "Pending transaction pool"},
+			{ID: txPoolLocal, Name: "Local transaction pool"},
+			{ID: txPoolPendingDiscard, Name: "Pending discard transaction pool"},
+			{ID: txPoolNofunds, Name: "Pool of transactions with no funds" },
+			{ID: txPoolPendingRatelimit, Name: "Pending transaction pool ratelimit"},
+			{ID: txPoolPendingReplace, Name: "Pending transaction pool to replace"},
+		},
```

Comment:
```suggestion
		Ctx: "geth.tx_pool_transactions", 
		Dims: Dims{
			{ID: txPoolInvalid, Name: "invalid"},
			{ID: txPoolPending, Name: "pending"},
			{ID: txPoolLocal, Name: "local"},
			{ID: txPoolPendingDiscard, Name: "pending_discard"},
			{ID: txPoolNofunds, Name: "no_funds" },
			{ID: txPoolPendingRatelimit, Name: "pending_ratelimit"},
			{ID: txPoolPendingReplace, Name: "pending_replace"},
		},
```

Reply:
The chart name is `Pending Transaction Pool`, however, there are some metrics w/o pending word 🤔 Are you sure? Do we want them in a single chart?

- txpool_invalid
- txpool_known (not in the chart)
- txpool_local
- txpool_overflowed (not in the chart)
- txpool_pending
- txpool_pending_discard
- txpool_pending_nofunds
- txpool_pending_ratelimit
- txpool_pending_replace

You didn't add 2 txpool metrics. Why?